bloom/generators/common.py

```diff
@@ -841,7 +841,7 @@ def _check_all_keys_are_valid(self, peer_packages, rosdistro):
                     all_keys_valid = False
         return all_keys_valid
 
-    def _pre_modify(self, key_unvalid_error_msg):
+    def check_all_keys_are_valid(self, key_unvalid_error_msg):
```

Comment:
I think `key_unvalid` is probably a typo of `key_invalid` that we can fix while we're here.

Reply:
Fixed in the newest commit